kitsune/tidings/models.py

```diff
@@ -0,0 +1,152 @@
+from django.conf import settings
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.sites.models import Site
+from django.db import connections, models, router
+
+from .utils import import_from_setting, reverse
```

Comment:
Nit: Perhaps for consistency with all of the other kitsune.tidings imports within this sub-package, we should make this an absolute import?
```python
from kitsune.tidings.utils import import_from_setting, reverse
```